yaml/pois.yaml

```diff
@@ -1812,3 +2106,32 @@ filters:
       <<: *output_properties
       kind: attraction
       tier: 6
+
+  # if a more specific shop isn't found, then default to generic shop kind
+  - filter:
+      shop: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: shop
+  # if a more specific craft isn't found, then default to generic craft kind
+  - filter:
+      craft: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: craft
+  # if a more specific off isn't found, then default to a generic office.
+  - filter:
+      office: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: office
```

Comment:
Please ensure test and documentation for `office` fallback.

Reply:
Added in dcce2ee.

Side note: Top `office` values we don't currently map to kinds:

* `energy_supplier`
* `logistics`
* `medical`
* `coworking`
* `forestry`
* `parish`
* `tax`
* `water_utility`
* `construction_company`
* `property_management`
* `charity`
* `guide`